completed/camunda-process-instances.js

```diff
@@ -0,0 +1,147 @@
+      ...formData.getHeaders()
+    },
+    data: formData
+  };
+
+  try {
+    const response = await axios(options);
+    const deployedResources = response.data.deployments;
+
+    // Emit deployed resources
+    deployedResources.forEach(x =>
+      console.log(
+        `Process Definition Key: ${x.processDefinition.processDefinitionKey}; Process Definition Id: ${x.processDefinition.processDefinitionId}`
+      )
+    );
+  } catch (error) {
+    // Emit an error from the server.
+    console.error(error.message);
+  }
+}
+
+// An action that creates and starts an instance of the specified process.
+async function createInstance([processDefinitionKey]) {
+  // Every request needs an access token.
+  const accessToken = await getAccessToken(authorizationConfiguration);
+
+  // These settings come from your .env file.
+  const camundaApiUrl = process.env.CAMUNDA_REST_ADDRESS;
+
+  // This is the API endpoint.
+  const url = `${camundaApiUrl}/process-instances`;
+
+  // Configure the API call.
+  const options = {
+    method: "POST",
+    url,
+    headers: {
+      Accept: "application/json",
+      Authorization: `Bearer ${accessToken}`
+    },
+    data: {
+      processDefinitionKey: processDefinitionKey,
+      variables: {
+        total: 90.0
+      }
+    }
+  };
+
+  try {
+    const response = await axios(options);
+    const processInstance = response.data;
+
+    // Emit new role to output.
+    console.log(`Process Instance Key: ${processInstance.processInstanceKey}`);
+  } catch (error) {
+    // Emit an error from the server.
+    console.error(error.message);
+  }
+}
+
+// An action that gets the process instance by the process instance key.
+async function viewInstance([processInstanceKey]) {
+  // Every request needs an access token.
+  const accessToken = await getAccessToken(authorizationConfiguration);
+
+  // These settings come from your .env file.
+  const camundaApiUrl = process.env.CAMUNDA_REST_ADDRESS;
+
+  // This is the API endpoint to get a specific role.
+  const url = `${camundaApiUrl}/process-instances/${processInstanceKey}`;
+
+  // Configure the API call.
+  const options = {
+    method: "GET",
+    url,
+    headers: {
+      Accept: "application/json",
+      Authorization: `Bearer ${accessToken}`
+    }
+  };
+
+  try {
+    // Call the endpoint.
+    const response = await axios(options);
+
+    // Process the results from the API call.
+    const results = response.data;
+
+    // Emit role to output.
+    console.log(
+      `Process instance name: ${results.processDefinitionName}; State: ${results.state};`
+    );
+  } catch (error) {
+    // Emit an error from the server.
+    console.error(error.message);
+  }
+}
+
+// These functions are aliased to specific command names for terseness.
+//   The name of each property translates to a method that can be called by the CLI.
+//   e.g. if we export a function named `list`, you can run `npm run cli zeebe get`.
```

Comment:
```suggestion
//   e.g. if we export a function named `list`, you can run `npm run cli processInstances get`.
```

(this is the name of this CLI, according to cli.js.)